runbot_buildout/models/runbot_build.py

```diff
@@ -414,7 +415,7 @@ def _adapt_buildout(self):
                 'options.without_demo = False\n'
                 'options.data_dir = %(datadir)s\n'
                 'options.dbfilter = %(dbfilter)s\n'
-                'options.lang = en_US' % dict(
+                'options.load_language = en_US' % dict(
```

Comment:
see https://github.com/OCA/runbot-addons/pull/187, but keep it here, then I'll close mine

Reply:
@hbrunn It got mixed up/in because my starting point was your branch. Now merged anyway.